observability/otel/otel-configuring-otelcol-metrics.adoc

```diff
@@ -4,11 +4,22 @@
 include::_attributes/common-attributes.adoc[]
 :context: otel-configuring-metrics
 
-//[id="setting-up-monitoring-for-otel"]
-//== Setting up monitoring for the {OTELShortName}
-//The {OTELOperator} supports monitoring and alerting of each OpenTelemtry Collector instance and exposes upgrade and operational metrics about the Operator itself.
+The OpenTelemetry Collector generates its own internal metrics to monitor its performance and health, including the following metrics:
```

Comment:
"including the following metrics:" reads like in addition to its own metrics. How about:

> The following list shows some of these metrics: